src/ui/components/filters/filterboxcomponent.js

```diff
@@ -308,10 +310,8 @@ export default class FilterBoxComponent extends Component {
    * Trigger a search with all filters in storage
    */
   _search () {
-    this.core.verticalSearch(this._config.verticalKey, {
-      setQueryParams: true,
-      resetPagination: true,
-      useFacets: true
-    });
+    const query = this.core.storage.get(StorageKeys.QUERY) || '';
```

Comment:
Now, FilterBox searches by setting a query trigger and query, instead of directly calling the verticalSearch method.
This is so, if somebody lands on a page with no search triggered on load, then triggers a search using a filter component, the query will be updated to "", which is what a search triggered by a static filter will use if there is no query set in storage

Reply:
This was an edge case with back navigation